### ISSUE TITLE
Release v1.36.8

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#2618]
:robot: A dedicated issue to run tests and deploy release [v1.36.8](https://github.com/deckhouse/deckhouse/milestone/113).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.36.8 | test-v1.36.8*` or a branch `main | release-1.36`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.20 | 1.21 | 1.22 | 1.23 | 1.24`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.36.8 | test-v1.36.8*` or a branch `main | release-1.36`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.21'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.20
/e2e/use/k8s/1.23

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/cri/containerd
/e2e/use/k8s/1.23

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.36
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [facca0](https://github.com/deckhouse/deckhouse/commit/facca0aa3725fa4bc3c0b02612f8dc03e7545c3a) in branch **main**:
```
[flant-integration] Fix erronous control-plane accounting in VMs (#2617)

Signed-off-by: Eugene Shevchenko <evgeny.shevchenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4289f5](https://github.com/deckhouse/deckhouse/commit/4289f5dc16892dd8ee4f80bb6075426294c2b814) in branch **main**:
```
Re-generate changelog v1.36.8 (#2623)
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.36.8** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [b9c554](https://github.com/deckhouse/deckhouse/commit/b9c554a261f33a505998763a35242f32a1b88038) in branch **main**:
```
Re-generate changelog v1.36.8 (#2626)
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.36.8** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [b90998](https://github.com/deckhouse/deckhouse/commit/b9099856173267897495f47f04b9a6677e967b96) in branch **main**:
```
[testing] e2e deckhouse ready timeout fix (#2630)

Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
Co-authored-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/build' command and started the workflow...
